src/main.cpp

```diff
@@ -34,7 +37,14 @@
 #include <GPIO.h>
 #include <Gyro_Acc.h>
 #include <I2CBus.h>
+#if IOEXT_ON == true
```

Comment:
It's ok, but it should be fine without `== true`, or another way of doing it is with 
````
#define ABC

#ifdef ABC
// do something here if ABC is defined
#endif
```

Reply:
Yes, can we do with #ifdef. The advantage of  #if ABC == flag is that you can switch it without comment out the #define ABC
In this special case I will remove it completly after migration is done.